_install-and-configure/install-opensearch/debian.md

```diff
@@ -159,12 +159,19 @@ APT, the primary package management tool for Debian–based operating systems, a
    sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> apt-get install opensearch={{site.opensearch_version}}
 
    # Use the following command for OpenSearch versions 2.11 and earlier:
-   sudo apt-get install opensearch={{site.opensearch_version}}
+   sudo apt-get install opensearch=<version>
```

Comment:
@natebower I don't know if this should include the version itself, or "<version>", 

Also line 148, should that be removed entirely, as you have to specify the version if you want older one anyway?